changelog/unreleased/pull-4100

```diff
@@ -0,0 +1,10 @@
+Enhancement: Make distribution package builds without self-update work
+
+Shortly after the self-update functionality and thus the `selfupdate` tag was
+introduced through PR #1949, the fix for Issue #2041 effectively turned the tag
```

Comment:
Although the historic details are somewhat interesting, they are too verbose for the changelog. Just state that builds always included the `selfupdate` feature even though it wasn't explicitly enabled at compile time.

Reply:
Thanks, how about the new text?